mne/time_frequency/tfr.py

```diff
@@ -505,7 +505,8 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     from ..viz.utils import _setup_vmin_vmax
 
     copy = baseline is not None
-    data = rescale(data, times, baseline, mode, copy=copy)
+    if mode is not None:
+        data = rescale(data, times, baseline, mode, copy=copy)
```

Comment:
Because it is not checked for ``None`` anywhere. You can try ``plot_stockwell`` on master and see why.